script.js

```diff
@@ -0,0 +1,108 @@
+let selectedLi;
+let target = '';
+let ul = document.querySelector('.page__levels');
+let button = document.querySelector('.page__button');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/EmilLachinov8/JS-project/pullRequest?prid=7662088)
```suggestion
let button = document.querySelector(".page__button");
```